### ISSUE TITLE
Comments could use inline word wrapping, or scroll frames.

### ISSUE THREAD

**Issue by cycomachead** [jmoenig/Snap--Build-Your-Own-Blocks#1671]
I was all excited to realize I could paste URLs (URLS!!) into my Snap! comments, like I do in, erm..., less pretty languages. However, URLs aren't so pretty in Snap<em>!</em>. 😢 

The first is what things looked like right after I pasted the URL, the second is after messing with the comment width.

![screen shot 2017-02-28 at 1 38 46 am](https://cloud.githubusercontent.com/assets/1505907/23399990/356388c4-fd57-11e6-9a75-aa34e518c6bb.png)

![screen shot 2017-02-28 at 1 39 13 am](https://cloud.githubusercontent.com/assets/1505907/23399996/3a95805e-fd57-11e6-954b-db255b0684a4.png)

**Comment by jmoenig** [comment]
I agree. I'd even go further and claim we want better word wrapping for all text elements, including speech balloons. Since we already have automatic word-wrapping for text elements, I think it might be time to simply enhance the existing mechanism to also include more smart ways to determine word delimiters aside from whitespace.

**Comment by brianharvey** [comment]
Why is this an "enhancement"?  The part about the text appearing way outside the comment box is a bug, isn't it?

**Comment by cycomachead** [comment]
> Why is this an "enhancement"? The part about the text appearing way outside the comment box is a bug, isn't it?

I guess it depends on how you define 'bug'. I was thinking that word wrapping is an 'enhancement', broader than just this case. Mostly it's ugly, but entirely what I'd expect given there's no word wrapping.

I suppose there _is_ a bug in that we don't auto-resize the width of a comment morph on paste. This one can go either way.

**Comment by brianharvey** [comment]
Huh?  This:
![foo](https://cloud.githubusercontent.com/assets/3885074/24069933/105a355a-0b71-11e7-8052-420d87da9b4f.png)
isn't a bad wrap, although I believe that that's the cause, but even if part of the URL is invisible off to the right of the comment, it's a bug that text appears _outside_ the box!
